client/view/Form/form.js

```diff
@@ -39,6 +40,7 @@ window.addEventListener('popstate', () => {
 })
 
 async function handleFormSubmit(e) {
+  e.preventDefault()
```

Comment:
@ariarzer 

I moved `e.preventDefault` it to the top of the `handleFormSubmit`.

Is it possible to do so? Or can we break something?